### PR TITLE
Change "File Size" Column name to "Size" in file browser

### LINKED ISSUE #16734: Rename "File Size" column in file browser to "Size"
## Description

It is possible to read file sizes when the file size column in the filebrowser is very narrow, so much so that the word "Size" isn't visible in the column header:

image

Shorten this column name to "Size" — there is no ambiguity here. 

For comparison, "Size" is used as a column header in macOS's Finder:

image

## Context

- Operating System and version: macOS Sonoma 14.6.1
- Browser and version: Firefox ESR 129.0.1
- JupyterLab version: 4.3.0beta1

### PR DESCRIPTION
**Pull request by ChairBorn**
## References

This PR fixes #16734 and changes the Name of the File Browser "File Size" column to "Size".

## Code changes

Line 2811 of [packages/filebrowser/src/listing.ts] changed to "Size"

## User-facing changes

Now users will see the column name as "Size"

Screenshot 2024-09-05 110521

There are other references to "File Size", for instance: 

line 1302 of [packages/filebrowser-extension/src/index.ts] - overlay Menu in file browser
line 947 /test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json
line 70 /test/jupyterlab/sidebars.test.ts

Let me know if you want me to change the above mentioned instances as well.

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @ChairBorn for working on this.

Would it be possible to make the column name responsive, similar to "Last Modified" / "Modified", as suggested in https://github.com/jupyterlab/jupyterlab/issues/16734#issuecomment-2328589765? So we can still show "File Size" if there is enough space, and just "Size" otherwise.

**Comment by ChairBorn** [comment]
Let me work on that

**Comment by krassowski** [comment]
Edited the top level comment to fix the issue number.

**Comment by ChairBorn** [comment]
I think it works now, let me know if you need any adjustments or fixes.

File: packages/filebrowser/src/listing.ts [hunk 1]
```diff
@@ -2808,7 +2808,11 @@ export namespace DirListing {
             small: trans.__('Modified'),
             large: trans.__('Last Modified')
           }),
-        file_size: () => this.createHeaderItemNode(trans.__('File Size')),
+        // file_size: () => this.createHeaderItemNode(trans.__('Size')),
```
Comment (krassowski) [inline packages/filebrowser/src/listing.ts:2811]: Let's remove the commented out code:

```suggestion
```

:)

Comment (ChairBorn) [reply packages/filebrowser/src/listing.ts:2811]: Thank you, was hiding in there 😂

Comment (ChairBorn) [reply packages/filebrowser/src/listing.ts]: Ready to close?

**Comment by JasonWeill** [comment]
@ChairBorn Thank you for opening this PR and for making these changes! @krassowski and I currently have a couple of open PRs that change the way the file browser's grid gets rendered: #16765 (Mike's) and #16751 (mine). We are likely to merge at least one of these in to resolve #16712, a launch blocker for 4.3.0. Because #16734 is not as time-sensitive, would it be possible to wait on this until #16712 gets closed?

**Comment by ChairBorn** [comment]
Sure thing! Let me know if you have any thing you need work on, I'll go through the issues list meanwhile and see if there is anything I'm able to work on.

**Comment by JasonWeill** [comment]
@ChairBorn Thank you for your contribution and your patience! #16765 was just merged in, which changes the file browser to use CSS grid. Could you please rebase to `main`, make any needed changes, and push to your branch? Thanks!

**Comment by ChairBorn** [comment]
@JasonWeill will do shortly

**Comment by ChairBorn** [comment]
> @ChairBorn Thank you for your contribution and your patience! #16765 was just merged in, which changes the file browser to use CSS grid. Could you please rebase to `main`, make any needed changes, and push to your branch? Thanks!

done, sorry traveling today, had to do it om the fly

**Comment by jtpio** [comment]
Thanks @ChairBorn.

It looks like the last merge picked up unrelated changes from the last release.

**Comment by ChairBorn** [comment]
> Thanks @ChairBorn.
> 
> It looks like the last merge picked up unrelated changes from the last release.
I'll clean those up

**Comment by JasonWeill** [comment]
If you back out your two most recent commits, your change looks a lot neater: https://github.com/jupyterlab/jupyterlab/pull/16749/files/c6f2608eea524c3b55121d274b76be78ec3c28ff

Could you try backing out your two most recent commits locally, then [rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) to the upstream `main` branch? At that point, running `git push -f` should forcibly update your remote branch.

**Comment by JasonWeill** [comment]
#16849 was created by checking out @ChairBorn 's main branch, backing up to the third most recent change, and rebasing to upstream `main`. I created it to run our tests on the modified, rebased code.

**Comment by jtpio** [comment]
Thanks @JasonWeill for helping with this, and @ChairBorn for your patience.

Maybe it would indeed be easier to continue with https://github.com/jupyterlab/jupyterlab/pull/16849. Looks like we could then merge https://github.com/jupyterlab/jupyterlab/pull/16849 once the CI is green.

**Comment by ChairBorn** [comment]
What's up guys, my bad, I'm not that great with proficient Git yet. Can you please tell me how it needs to be done? Appreciate you both bearing with me. 

Thanks @JasonWeill @jtpio. Let me know what needs to be done on my end, I messed up switching my devices yesterday and pollinated a dirty tree in....

**Comment by JasonWeill** [comment]
Try reverting the two most recent commits with `git revert HEAD~2..HEAD`, then rebase with `git rebase upstream/main` (assuming that you have a remote called `upstream` that points to `git@github.com:jupyterlab/jupyterlab.git` or `https://github.com/jupyterlab/jupyterlab.git`). You can find a list of your remotes with `git remote -v`. Once you have finished rebasing, you can run `git push -f` to forcibly update your personal branch, and this pull request will update.

Or, we can just merge in #16849, which has the same change and is rebased from `main` already. Thanks again for your work, @ChairBorn !

**Pull request closed without merging**